### ISSUE TITLE
test errors under django > 1.2.3

### ISSUE THREAD

**Issue by slinkp** [openplans/openblock#119]
Django 1.2.3 gives 5 failures, 6 errors in the test suite.
Django 1.2.4 or 1.2.5 gives 4 failures, 51 errors.

```
(20110106)pw@slinktopp2 openblock $ easy_install --upgrade django==1.2.3 > /dev/null 2>&1 ; django-admin.py test 2>&1 | grep failures
FAILED (failures=5, errors=6)
(20110106)pw@slinktopp2 openblock $ easy_install --upgrade django==1.2.4 > /dev/null 2>&1 ; django-admin.py test 2>&1 | grep failures
FAILED (failures=4, errors=51)
(20110106)pw@slinktopp2 openblock $ easy_install --upgrade django==1.2.5 > /dev/null 2>&1 ; django-admin.py test 2>&1 | grep failures
FAILED (failures=4, errors=51)
(20110106)pw@slinktopp2 openblock $ easy_install --upgrade django==1.2.3 > /dev/null 2>&1 ; django-admin.py test 2>&1 | grep failures
FAILED (failures=5, errors=6)

```

**Comment by slinkp** [comment]
(In [a521db5ed492045d0de39ed9d970f8547cebacb5]) Nail down the django requirement. refs #119

**Comment by slinkp** [comment]
Fixed in [171347b65fab438b79d105609cb1bac38f3a5b4e] by removing multi-db support.

**Comment by slinkp** [comment]
Ticket imported from Trac:
 http://developer.openblockproject.org/ticket/119
Reported by: slinkp
